code/modules/urist/verbs/antagooc.dm

```diff
@@ -1,10 +1,11 @@
-var/global/normal_aooc_colour = "#FF0000"
+var/global/normal_aooc_color = "#FF3333" //Screw british speling of color. COLOR is correct. Like math. MATH is correct. MATHS is stoopid
 
 /client/verb/aooc(msg as text)
 	set name = "AOOC"
 	set category = "OOC"
+	//set hidden = 1
```

Comment:
I may be able to find something later, so I kept it. For the moment though, I am stumped